CHANGES.rst

```diff
@@ -10,6 +10,7 @@ Bug Fixes
 
 - Fixed bug on MOSVIZ where an exception was raised when loading JWST S2D file from a directory. 
 
+
```

Comment:
there should be a changelog entry here pointing to the original PR (#3547)

Reply:
there is

Reply:
So the changelog entry got backported separately, but not the actual code?

Reply:
im guessing it snuck in with a conflict resolution in the file on one of the other backport PRs